Lib/zoneinfo/__init__.py

```diff
@@ -0,0 +1,29 @@
+__all__ = [
+    "ZoneInfo",
+    "reset_tzpath",
+    "TZPATH",
+    "ZoneInfoNotFoundError",
+    "InvalidTZPathWarning",
+]
+
+from . import _tzpath
+from ._common import ZoneInfoNotFoundError
+
+try:
+    from _czoneinfo import ZoneInfo
+except ImportError:  # pragma: nocover
+    from ._zoneinfo import ZoneInfo
+
+reset_tzpath = _tzpath.reset_tzpath
+InvalidTZPathWarning = _tzpath.InvalidTZPathWarning
+
+
+def __getattr__(name):
+    if name == "TZPATH":
+        return _tzpath.TZPATH
+    else:
+        raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
+
+def __dir__():
+    return sorted(__all__ + ["__version__"])
```

Comment:
This seems to be excluding other stuff like `zoneinfo.__path__`. Any suggestions there?

Reply:
`sorted(list(globals()) + ['__TZPATH__'])`?

The `__version__` looks extraneous.

Reply:
Ah, good call. `__version__` only makes sense in the backport.

The `list(globals())` part solves the other problem, too. Thanks!